packages/analytics-js/package.json

```diff
@@ -35,18 +43,19 @@
   "homepage": "https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md",
   "scripts": {
     "clean": "rimraf -rf ./dist && rimraf -rf ./node_modules/.cache && rimraf tsconfig.build.tsbuildinfo",
-    "start": "ONLY_IIFE=true rollup --config rollup-configs/rollup.config.mjs --watch --environment DEV_SERVER,PROD_DEBUG",
+    "start": "ONLY_IIFE=true rollup --config --watch --environment DEV_SERVER,PROD_DEBUG",
     "build": "npm run build:browser && npm run build:package",
     "build:browser:modern": "BROWSERSLIST_ENV=modern npm run build:browser",
-    "build:browser": "rollup --config rollup-configs/rollup.config.mjs --environment VERSION:$npm_package_version,UGLIFY,PROD_DEBUG,STAGING:$npm_config_staging,ENV:prod",
-    "build:npm": "rollup --config rollup-configs/rollup.config.mjs --environment VERSION:$npm_package_version,ENV:prod",
-    "build:npm:modern": "BROWSERSLIST_ENV=modern rollup --config rollup-configs/rollup.config.mjs --environment VERSION:$npm_package_version,ENV:prod",
-    "build:worker": "exit 0",
-    "build:package": "npm run build:npm && npm run build:worker",
-    "test": "ONLY_IIFE=true npm run build:browser && sed 's/var rudderanalytics/rudderanalytics/g' dist/legacy/iife/index.js > __mocks__/cdnSDKv3.js && jest --watchAll=false --detectOpenHandles --forceExit --maxWorkers=50% --coverage",
-    "test:ci": "ONLY_IIFE=true npm run build:browser && sed 's/var rudderanalytics/rudderanalytics/g' dist/legacy/iife/index.js > __mocks__/cdnSDKv3.js && jest --watchAll=false --detectOpenHandles --forceExit --maxWorkers=2 --coverage",
+    "build:browser": "rollup --config --environment VERSION:$npm_package_version,UGLIFY,PROD_DEBUG,ENV:prod",
+    "build:npm": "rollup --config --environment VERSION:$npm_package_version,ENV:prod",
+    "build:npm:modern": "BROWSERSLIST_ENV=modern npm run build:npm",
+    "build:package": "npm run build:npm:modern && npm run build:npm",
```

Comment:
The NPM package build commands are also building the CDN artifacts. Perhaps, we should differentiate this with an env.

Reply:
Moreover, it generates unminified files.

Reply:
not sure I am following, build:package build 2 folders one with the legacy export fo the package (that included the plugins) and one with the modern (thatwill lead plugins from CDN as federated modules)

we do not minify, as these are npm modules, if the consumer wants to minify their bundle it will be in their build consiguration for their application bundling. Easier to debug locally ass develoer if the npm package is not minified.

Reply:
These commands also build the CDN artifacts of the core SDK apart from the NPM bundles. I was expecting these commands will generate the bundle artefacts just need for publishing the NPM package.

> Moreover, it generates unminified files.
This is for the core SDK CDN artifacts these commands generate.

Reply:
got it, lt me clean up

Reply:
done